bitnami/consul/templates/consul-service.yaml

```diff
@@ -26,5 +29,10 @@ spec:
     - name: consuldns-udp
       protocol: "UDP"
       port: {{.Values.service.consulDnsPort}}
+{{- if .Values.metrics.enabled }}
+    - name: metrics
+      port: 9104
```

Comment:
Adding the property `.Values.metrics.service.` would also allow to provide with a parameter named `port` to customize it. Nonetheless, the default `podAnnotations.prometheus.io/port` is set to `9107` instead of `9104`, so be careful.